Mathport.lean

```diff
@@ -43,10 +43,11 @@ def mathport1 (config : Config) (path : Path) : IO Unit := do
       }
       let cmdState : Elab.Command.State := Lean.Elab.Command.mkState (env := env) (opts := opts)
 
+      let commitInfo ← IO.getEnv "MATHPORT_COMMIT_INFO"
```

Comment:
Can't this go in the `config`?

Reply:
Ah yes it could (I assume you just mean adding it to the `config` variable here, and still using the environment variable hack to get the data here in the first place).

Reply:
You can also put the data in the config in the first place, I think, using something like the config.oneshot.json makefile rule to add the data to the config before running mathport. I don't really like the idea of making the `./build/bin/mathport` invocation line any more complicated than it already is.

Reply:
Oh, I guess making a new `config.[lean/mathlib].json` file would work. But I don't have `jq` installed on my office machine so I'm unable to implement this :)